.github/workflows/pages-deployment.yml

```diff
@@ -38,7 +38,7 @@ jobs:
         uses: cloudflare/pages-action@c2ad89679f0dee1ff4bcf9b8bbe1cf52d176cb29  # v1.3.0
         with:
           accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
```

Comment:
I changed CLOUDFLARE_ACCOUNT_ID to CLOUDFLARE_API_TOKEN.

Did I missunderstand the brief?

Reply:
I think `CLOUDFLARE_ACCOUNT_ID` should stay as is.

Reply:
Or, for the most clarity, what's required here are organisation secrets in both `opensafely` (for this repository) and `opensafely-core` (for `databuilder`) with the names:

* `CLOUDFLARE_ACCOUNT_ID` — as it was in this repository, our Cloudflare account's ID
* `CLOUDFLARE_DIRECT_UPLOAD_API_TOKEN` is a new name but with the same value as `CLOUDFLARE_API_TOKEN` in this repository

Reply:
Ok, I think we just need to rename CLOUD_FLARE_API_TOKEN to CLOUDFLARE_ACCOUNT_ID, as I misread the original suggestions

Reply:
I've renamed `s/CLOUDFLARE_API_TOKEN/CLOUDFLARE_ACCOUNT_ID/` in both orgs and in bitwarden.

Reply:
I've delete CLOUDFLARE_API_TOKEN from both org secrets, so it should just be this repos version that needs removing.

Reply:
@bloodearnest: thanks!